backend/compact-connect/lambdas/provider-data-v1/tests/unit/test_data_model/test_schema/test_license.py

```diff
@@ -8,13 +8,13 @@
 
 class TestLicenseSchema(TstLambdas):
     def test_validate_post(self):
-        from data_model.schema.license import LicensePostSchema
+        from cc_common.data_model.schema.license import LicensePostSchema
```

Comment:
I would, but I've already eaten my allocated time for this particular effort. If I do make this refactor, it would be the very last thing to address given time constraints.

Reply:
Ok, looking back at the AC you're trying to meet, we could call moving these tests as out of scope to not hold up this PR. We'll definitely want to make a point to continue refactoring tests, etc. so that they make sense with the new layout as we go forward.

Reply:
I've pulled in the common schema tests, there were a couple tests that relied on handler logic within the test setup, so I wasn't able to extract those, but its much better than what it was.